pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
 	OpenShiftSharedResourcesNamespace string
 }
 
-func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
+func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
```

Comment:
fullName is the name someone access you as in a larger context.  Name is your name.  

You are David.  Your full name is David Eads.

----- Original Message -----
> > @@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
> >  	OpenShiftSharedResourcesNamespace string
> >  }
> >  
> > -func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
> > +func NewCommandCreateBootstrapPolicyFile(commandName string, baseName
> > string, out io.Writer) *cobra.Command {
> 
> Seems like we'd want to pass name and baseName as opposed to name and
> fullName.  Why would we want fullName instead?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1454/files#r27217531